docs/checks/internal/checks.yaml

```diff
@@ -204,17 +271,25 @@ checks:
   Contributors:
     risk: Low
     tags: source-code
-    short: Determines if the project has a set of contributors from multiple companies.
+    short: Determines if the project has a set of contributors from multiple organizations (e.g., companies).
     description: >-
       This check tries to determine if the project has a set of contributors from
-      multiple companies.
+      multiple organizations (e.g., companies).
 
       Low score has 'Low' risk.
 
+      Some projects cannot meet this requirement.
+      Obviously projects with only one active participant
+      cannot meet it.
+      Many projects have a narrow scope and may not be able to attract
+      the interest of multiple organizations.
+      See [Code Reviews](#code-reviews) for added notes about
+      projects with a small number of participants.
+
       The check works by looking at the authors of recent commits
       and checking the `Company` field on the GitHub user profile. A contributor
       must have at least 5 commits in the last 30 commits. The check succeeds if
```

Comment:
note: you can remove the part about when the check succeeds. We don't use Pas/Fail anymore.

Reply:
Ok, will do.